build.assets/Dockerfile-centos7-fips

```diff
@@ -0,0 +1,73 @@
+FROM centos:7
+
+ENV LANGUAGE=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    LC_ALL=en_US.UTF-8 \
+    LC_CTYPE=en_US.UTF-8
+
+ARG RUST_VERSION
+ARG BORINGCRYPTO_RUNTIME
+ARG GO_BOOTSTRAP_RUNTIME=go1.9.7
+
+ENV LANGUAGE=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    LC_ALL=en_US.UTF-8 \
+    LC_CTYPE=en_US.UTF-8
```

Comment:
Duplicate of line 3
```suggestion
```

Reply:
Thanks Tim. I didn't mean for you to get tagged on this just yet. I thought marking the PR as draft would do that. Anyway, I will fix this and re-request a review when I'm a bit further along.

Reply:
Yeah sorry I realized it was still a draft after I commented. For some reason the bot requested me to review it last night.